src/test/java/example/TraverseDemoTest.java

```diff
@@ -0,0 +1,57 @@
+package example;
+
+import org.junit.jupiter.api.*;
+import org.neo4j.driver.*;
+import org.neo4j.harness.Neo4j;
+import org.neo4j.harness.Neo4jBuilders;
+
+import java.io.File;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class TraverseDemoTest {
+
+    private static final Config driverConfig = Config.builder().withoutEncryption().build();
+    private Neo4j embeddedDatabaseServer;
+
+    @BeforeAll
+    void initializeNeo4j() {
+        this.embeddedDatabaseServer = Neo4jBuilders.newInProcessBuilder()
+                .withProcedure(TraverseDemo.class)
+                .withFixture(new File(getClass().getResource("/movie.cypher").getPath()))
+                .build();
```

Comment:
```suggestion
        var sw = new StringWriter();
        try (var in = new BufferedReader(new InputStreamReader(getClass().getResourceAsStream("/movie.cypher")))) {
            in.transferTo(sw);
            sw.flush();
        }

        this.embeddedDatabaseServer = Neo4jBuilders.newInProcessBuilder()
            .withProcedure(TraverseDemo.class)
            .withFixture(sw.toString())
            .build();
```

Don't assume a resource being a file. I should we should go with a good example here.